examples/favorites_advanced/bloc_favorites_advanced/lib/feature_puppy/edit/ui_components/puppy_edit_card.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
```

Comment:
@StanislavPrime, the same comment as above

Reply:
We have the same widget `PuppyEditCard`, but it is in the `rx_bloc_favorites_advanced `and not in the `base `project . It is here: https://github.com/Prime-Holding/rx_bloc/blob/develop/examples/favorites_advanced/rx_bloc_favorites_advanced/lib/feature_puppy/edit/ui_components/puppy_edit_card.dart
But I can move it in the base project and use it from there.

Reply:
The widget was replaced with the PuppyEditCard widget from the base project